CHANGELOG.md

```diff
@@ -7,122 +7,74 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
+- Swaps: reduce the default slippage from 3% to 2% ([#14863](https://github.com/MetaMask/metamask-extension/pull/14863))
+- Swaps: update STX status content page ([#14805](https://github.com/MetaMask/metamask-extension/pull/14805))
+  - Change STX status timer description dynamically based on the stxStatusDeadline param
+  - Change "Swap complete in < X:XX" to "Swap will complete in < X:XX"
+  - Change "Privately submitting the Swap..." to "Privately submitting your Swap..."
+- Swaps: migrate the estimateGas API call to getFees for STX for reducing API calls ([#14767](https://github.com/MetaMask/metamask-extension/pull/14767))
+- Swaps: disable STX if a regular tx is in progress ([#14554](https://github.com/MetaMask/metamask-extension/pull/14554))
+- Swaps: STX improvements ([#14622](https://github.com/MetaMask/metamask-extension/pull/14622))
+  - Track real fees in ETH for approval and trade txs in the Swap Completed event
+  - Use USDConversionRate for calculating estimated fees in USD
+  - Fix a condition for checking non-prod `METAMASK_ENVIRONMENT`
+  - Load `parameters.stxMaxFeeMultiplier` from network config params
+- Swaps: remove gas editing ([#14673](https://github.com/MetaMask/metamask-extension/pull/14673))
+
+### Fixed
+- Fix line-wrapping in Edit Nickname screen by decreasing the padding ([#14842](https://github.com/MetaMask/metamask-extension/pull/14842))
+- Fix permissions to https://chainid.network/chains.json by adding the domain into the manifest.json file ([#14810](https://github.com/MetaMask/metamask-extension/pull/14810))
+- Fix switching between ETH and USD, so the amount value is updated correctly ([#13827](https://github.com/MetaMask/metamask-extension/pull/13827))
+- Fix add recipient flow issue with contact list/recent recipients ([#14771](https://github.com/MetaMask/metamask-extension/pull/14771))
+- Fix lock button sizing for text exceeding button boundaries ([#14335](https://github.com/MetaMask/metamask-extension/pull/14335))
+- Fix all "MetaMask" instances wrongly written as "Metamask" 
+  - ([#14851](https://github.com/MetaMask/metamask-extension/pull/14851))
+  - ([#14848](https://github.com/MetaMask/metamask-extension/pull/14848))
+- Fix design break on the Settings Navbar when Internationalization ([#14012](https://github.com/MetaMask/metamask-extension/pull/14012))
+- Fix missing white spacing for keystone transaction qrcode in darkmode ([#14798](https://github.com/MetaMask/metamask-extension/pull/14798))
+- Fix ledger accounts issue by upgrading `@metamask/eth-ledger-bridge-keyring` ([#14799](https://github.com/MetaMask/metamask-extension/pull/14799))
+- Fix proptype errors in network dropdown, tx list item details, and account details modal tests ([#14747](https://github.com/MetaMask/metamask-extension/pull/14747))
+- Fix wrong tx type on edit, by updating tx type in case it changed ([#14721](https://github.com/MetaMask/metamask-extension/pull/14721))
```

Comment:
Suggestion to change "Fix wrong tx type on edit, by updating tx type in case it changed" to "Fix bug that could cause the incorrect token to be selected after editing the token via the send edit flow"

Reply:
noted. Ty